src/test/java/com/commercetools/sync/commons/helpers/BaseSyncStatisticsTest.java

```diff
@@ -94,24 +95,24 @@ public void getStatisticsAsJSONString_WithoutCalculatingProcessingTime_ShouldGet
         baseSyncStatistics.incrementUpdated();
         baseSyncStatistics.incrementProcessed();
 
-        final String statisticsAsJsonString = getStatisticsAsJSONString(baseSyncStatistics);
+        final String statisticsAsJsonString = getStatisticsAsJsonString(baseSyncStatistics);
         assertThat(statisticsAsJsonString)
-                .isEqualTo("{\"reportMessage\":\"Summary: 3 categories were processed in total (1 created, 1 updated and" +
-                        " 1 categories failed to sync).\",\"" +
-                        "updated\":1,\"" +
-                        "created\":1,\"" +
-                        "failed\":1,\"" +
-                        "processed\":3,\"" +
-                        "processingTimeInDays\":" + baseSyncStatistics.getProcessingTimeInDays() + ",\"" +
-                        "processingTimeInHours\":" + baseSyncStatistics.getProcessingTimeInHours() + ",\"" +
-                        "processingTimeInMinutes\":" + baseSyncStatistics.getProcessingTimeInMinutes() + ",\"" +
-                        "processingTimeInSeconds\":" + baseSyncStatistics.getProcessingTimeInSeconds() + ",\"" +
-                        "processingTimeInMillis\":" + baseSyncStatistics.getProcessingTimeInMillis() + ",\"" +
-                        "humanReadableProcessingTime\":\"" + baseSyncStatistics.getHumanReadableProcessingTime() + "\"}");
+            .isEqualTo("{\"reportMessage\":\"Summary: 3 categories were processed in total (1 created, 1 updated and"
+                + " 1 categories failed to sync).\",\""
+                + "updated\":1,\""
+                + "created\":1,\""
+                + "failed\":1,\""
+                + "processed\":3,\""
+                + "processingTimeInDays\":" + baseSyncStatistics.getProcessingTimeInDays() + ",\""
+                + "processingTimeInHours\":" + baseSyncStatistics.getProcessingTimeInHours() + ",\""
+                + "processingTimeInMinutes\":" + baseSyncStatistics.getProcessingTimeInMinutes() + ",\""
+                + "processingTimeInSeconds\":" + baseSyncStatistics.getProcessingTimeInSeconds() + ",\""
+                + "processingTimeInMillis\":" + baseSyncStatistics.getProcessingTimeInMillis() + ",\""
+                + "humanReadableProcessingTime\":\"" + baseSyncStatistics.getHumanReadableProcessingTime() + "\"}");
```

Comment:
these test that test manually the numbers are already there in the test class above this one. However, this specific unit test is to test the conversion of the Statistics object when it's converted to a JSON string.